horreum-backend/src/main/resources/db/changeLog.xml

```diff
@@ -4090,4 +4089,32 @@
             CREATE TRIGGER before_variable_delete BEFORE DELETE ON variable FOR EACH ROW EXECUTE FUNCTION before_variable_delete_func();
         </sql>
     </changeSet>
+    <changeSet id="107" author="stalep">
+        <createSequence sequenceName="actionlog_seq" startValue="1" incrementBy="50" cacheSize="1" />
```

Comment:
Are all of these sequences new? will we still get contiguous id's for the existing sequences?

Reply:
Yes, they are all new. Regarding your last question idk.